Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1870,6 +1885,47 @@ RefPtr<Gfx::Font const> StyleComputer::compute_font_for_style_values(DOM::Elemen
         found_font = find_font(font_family.as_custom_ident().custom_ident());
     }
 
+    // 5.2.6 If no matching face exists or the matched face does not contain a glyph for the character to be
+    // rendered, the next family name is selected and the previous three steps repeated. Glyphs from other
+    // faces in the family are not considered. The only exception is that user agents may optionally substitute
+    // a synthetically obliqued version of the default face if that face supports a given glyph and synthesis
+    // of these faces is permitted by the value of the font-synthesis property. For example, a synthetic italic version
+    // of the regular face might be used if the italic face doesn’t support glyphs for Arabic.
+    auto face_contains_glyphs = [&](RefPtr<Gfx::Font const> font, FlyString const& text) -> bool {
+        if (font) {
+            for (auto code_point : text.code_points()) {
+                if (!is_ascii_control(code_point) && !font->contains_glyph(code_point))
+                    return false;
+            }
+            return true;
+        }
+        return false;
+    };
+    if (element) {
+        if (auto text = element->child_text_content(); !text.is_empty()) {
+            auto text_content = String::from_deprecated_string(text);
```

Comment:
```suggestion
            auto text_content = MUST(String::from_deprecated_string(text));
```
`String::from_deprecated_string()` can only fail for two reasons:
1. OOM.
2. Invalid UTF-8 text.

We don't care about 1, and 2 can't happen because the HTML parser ensures that the text is UTF-8 already.

Reply:
I have updated to use the MUST.